client/src/main/java/io/pravega/client/connection/impl/RawClient.java

```diff
@@ -80,6 +83,18 @@ public void authTokenCheckFailed(WireCommands.AuthTokenCheckFailed authTokenChec
             log.warn("Auth token check failed on segment {} with {}", segmentId, authTokenCheckFailed);
             closeConnection(new AuthenticationException(authTokenCheckFailed.toString()));
         }
+
+        @Override
+        public void errorMessage(WireCommands.ErrorMessage errorMessage) {
```

Comment:
It should be enough to implement this once on `FailingReplyProcessor`; if you observe - all the classes where you overrode this method do more or less the same thing. You can just write it once.